packages/story-editor/src/components/canvas/displayElement.js

```diff
@@ -240,6 +241,7 @@ function DisplayElement({
             isCurrentResourceUploading={isCurrentResourceUploading}
             maybeEnqueueFontStyle={maybeEnqueueFontStyle}
             siblingCount={siblingCount}
+            renderResourcePlaceholder={renderResourcePlaceholder}
```

Comment:
Since this seems to be used only in the `element-library` package and is only for the media elements, perhaps the util and the usage could be moved to the `element-library` directly to the corresponding elements' display logic? Seems like something that could belong directly to the element display.

Or is it here because `renderResourcePlaceholder` uses the `react-blurhash` library, and it shouldn't be imported from the `element-library` package?
Thoughts?

Reply:
> Or is it here because `renderResourcePlaceholder` uses the `react-blurhash` library, and it shouldn't be imported from the `element-library` package?

Correct. See discussion at https://github.com/GoogleForCreators/web-stories-wp/issues/10258#issuecomment-1171408347